### PR TITLE
road home nerf

### PR DESCRIPTION
**Pull request by Caldony**
## About The Pull Request

Nerfs road home to be a little a slower and only hits people with her AOE if she can see them.
I also changed her records from 02 to 01 since I mixed animal and human.

## Why It's Good For The Game

Road home has an instant insanity and insane CC. Despite the fact she's rather easy to kill, it should at least be a lot easier to dodge her status effect considering how big the effect actually is, especially since she's a HE.

## Changelog
:cl:
balance: road home is now slower and only hits people in view with her flip.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/abnormality/he/road_home.dm [hunk 1]
```diff
@@ -140,7 +140,7 @@
 		return
 
 	playsound(src, 'sound/abnormalities/roadhome/House_NormalAtk.ogg', 100, FALSE, 8)
-	for(var/mob/living/L in livinginrange(15, src))
+	for(var/mob/living/L in livinginview(15, src))
```
Comment (Kitsunemitsu) [inline code/modules/mob/living/simple_animal/abnormality/he/road_home.dm:143]: Please use In View, Living in view takes all the mobs on the Z level gets their distance to target, and should only really be used about 50

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/simple_animal/abnormality/he/road_home.dm (hunk 1 above)
Comment (Caldony) [reply code/modules/mob/living/simple_animal/abnormality/he/road_home.dm]: We should probably change rudolta then, fixed.

**Review by Kitsunemitsu — APPROVED** [review]
Changes are good.

**Review by Kitsunemitsu — APPROVED** [review]
LGTM

**Pull request merged**